traits/traits.rs

```diff
@@ -99,6 +110,14 @@ impl From<regex::Error> for Error {
     }
 }
 
+impl From<gimli::Error> for Error {
```

Comment:
(This is going to need the `cfg` too -- sorry I forgot to point it out earlier!)

Reply:
No problem! Pardon, I should've seen that too. Fixing that now!